scanapp/models.py

```diff
@@ -108,29 +110,39 @@ class License(models.Model):
     """
     Represent a license as detected in a file
     """
+
     def __str__(self):
         return self.key
 
     scanned_file = models.ForeignKey(ScannedFile)
     key = models.CharField(max_length=200, help_text='Key of license')
     score = models.IntegerField(help_text='Score of license')
-    short_name = models.CharField(max_length=200, help_text='Short name of the license')
```

Comment:
I am not too keen on this type of strict PEP8 line wrapping for the sake of it. I consider that lines should be ~ 80ish in length. Here you are reformatting readable code in something that is no longer readable. What's the point?

Reply:
@pombredanne If you wish to increase the number of characters allowed per line, coala has a feature for that too. We can define ` max_line_length ` according to our need. 😄

Reply:
I did not see any of the proposed line breakings that made the code more beautiful and easier to read.

Reply:
Okay if it isn't much of a use, then the better alternative I guess is to should drop the idea of reformatting. your views?

Reply:
Formatting is fine, as long as we have the right rules such as line length in the 80ish range. Skip migrations, etc...

Reply:
It looks like this repo should have `max_line_length = 100`.  There should only be a few instances of violations of that limit.
Then it will be easier to review and consider those violations.

If there are still more than 20 violations of `max_line_length = 100` , increase it to `max_line_length = 120`.
The goal of adding a `.coafile` is _not_ to fix all the problems, but instead it is to prevent future regressions making the codebase style even less unified.
After the `.coafile` is merged and being checked by CI, additional improvements can be made to style rules incrementally.

Reply:
Okay will do it asap :+1: